### PR TITLE
piparser: Add piparser disable flag

### LINKED ISSUE #1374: need option to disable politeia code
On simnet, the politeia code makes no sense.  The following should be skipped along with all other Pi operations:

```
[INF] DATD: Setting up the Politeia's proposals clone repository. Please wait...
```

### PR DESCRIPTION
**Pull request by dmigwi**
A new flag to disable Piparser and Politea API query is added.

 `--disable-piparser` ensures that `/proposals` and `/proposal` page returns:

Screenshot from 2019-06-07 15-04-50

The API returns:
```bash
$ curl http://localhost:7777/api/proposal/a4f2a91c8589b2e5a955798d6c0f4f77f2eec13b62063c5f4102c21913dcaf32 
piparser is disabled.
```

Fixes #1374

### CODE REVIEW

File: db/dbtypes/go.sum [hunk 1]
```diff
@@ -149,6 +150,7 @@ github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORN
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
+github.com/lib/pq v1.0.0/go.mod h1:5WUZQaWbwv1U+lTReE5YruASi9Al49XbQIvNi/34Woo=
```
Comment (chappjc) [inline db/dbtypes/go.sum]: We'll be updating all the module file shortly, so let's revert these changes for now.

**Comment by chappjc** [comment]
Sorry for the conflict.  Just need to have Politeia capitalized in config.go on those two lines.

**Comment by dmigwi** [comment]
I could add the automatic disabling while on simnet functionality.

File: db/dbtypes/go.sum (hunk 1 above)
Comment (dmigwi) [reply db/dbtypes/go.sum]: Seems like `github.com/decred/dcrdata/db/dcrpg` has an older version of that package that won't disappear unless I update package `dcrpg` too.

Comment (chappjc) [reply db/dbtypes/go.sum]: It's because of the one "blackbox" test file.  No worries.
